README.md

```diff
@@ -56,12 +56,12 @@ Durable sync needs only a valid mysql DSN in order to set up connections to a My
 
 To setup a fresh MySQL DB and user: (`mysql -u root`):
 
-    ```
```

Comment:
Minor nit: unrelated changes?. No big deal for now, we can leave as is, but noting for the future as more changes mean more for folks to review.

Reply:
Sorry! Got your point.

Reply:
no worries at all